statusnet/library/twitteroauth.php

```diff
@@ -60,60 +73,49 @@ function requestTokenURL()
 		return 'https://api.twitter.com/oauth/request_token';
 	}
 
-	/**
-	 * Debug helpers
-	 */
-	function lastStatusCode()
-	{
-		return $this->http_status;
-	}
-
-	function lastAPICall()
-	{
-		return $this->last_api_call;
-	}
-
-	/**
-	 * construct TwitterOAuth object
-	 */
-	function __construct($consumer_key, $consumer_secret, $oauth_token = NULL, $oauth_token_secret = NULL)
+	function __construct($consumer_key, $consumer_secret, $oauth_token = null, $oauth_token_secret = null)
 	{
 		$this->sha1_method = new OAuthSignatureMethod_HMAC_SHA1();
 		$this->consumer = new OAuthConsumer($consumer_key, $consumer_secret);
 		if (!empty($oauth_token) && !empty($oauth_token_secret)) {
-			$this->token = new OAuthConsumer($oauth_token, $oauth_token_secret);
+			$this->token = new OAuthToken($oauth_token, $oauth_token_secret);
 		} else {
-			$this->token = NULL;
+			$this->token = null;
 		}
 	}
 
 	/**
-	 * Get a request_token from Twitter
+	 * Get a request_token from Tumblr
```

Comment:
In fact this isn't Tumblr, it is the Twitter compatible API from GNU Social.

Reply:
Good catch. I copy-pasted the fixed file from the Tumblr extension to fix Twitter's.

Reply:
Was this your only "real" change to the tumblr part? Since the indentation changed, the real changes are hard to catch.

Reply:
You can disable the match on whitespaces in the Github diff. I've improved the phpDoc and remove unused stuff but this is the only significant change in both files.